src/v3/src/components/PasswordRequirements/PasswordMatches.tsx

```diff
@@ -61,6 +61,7 @@ const PasswordMatches: UISchemaElementComponent<{
       <List
         id="credentials.newPassword-list"
         dense
+        disablePadding
```

Comment:
There was some extra padding (caused by the MUI list component), thought it looked oddly spaced, so disabled here (which also matches the padding in the Password requirements list above)